roles/common/tasks/monitoring.yml

```diff
@@ -225,3 +225,6 @@
     check: "{{ sensu_checks.common.check_raid }}"
   notify: restart sensu-client missing ok
   when: common.hwraid.enabled|bool
+
+- include: monitoring-vyatta.yml
+  hosts: controller
```

Comment:
is there maybe a way to check something like "when: inventory_hostname in groups['controller'] and groups['vyatta-*'] exists"?